eras/alonzo/impl/src/Cardano/Ledger/Alonzo/PlutusScriptApi.hs

```diff
@@ -24,27 +26,24 @@ module Cardano.Ledger.Alonzo.PlutusScriptApi (
 )
 where
 
-import Cardano.Ledger.Alonzo.Core (MaryEraTxBody (..))
+import Cardano.Ledger.Alonzo.Core hiding (TranslationError)
```

Comment:
I personally prefer to avoid open imports. But if I'm the only one, I won't object.

Reply:
IIRC we agreed to allow open imports for `Core` libraries

Reply:
right, I do remember us saying that `Cardano.Ledger.Core` might be a reasonable middle ground. Let's try all the `Core` libraries and see how it goes.

Reply:
It might even be better to only allow open importing the Core module of the current era, since it will re-export all the previous Core modules and we also have better control over what not to export should there be something that clashes with things in that era.